conf/firmwares/subsystems/shared/baro_board.makefile

```diff
@@ -177,6 +177,17 @@ else ifeq ($(BOARD),$(filter $(BOARD),px4fmu))
   BARO_BOARD_SRCS += peripherals/ms5611_spi.c
   BARO_BOARD_SRCS += boards/baro_board_ms5611_spi.c
 
+# PX4FMU_4.0 a.k.a. PX4_PIXRACER
+else ifeq ($(BOARD),$(filter $(BOARD),px4fmu_4.0))
```

Comment:
instead of checking for BOARD = px4fmu_4.0, you should only check for BOARD = px4fmu and distinguish between the board version in the general BOARD = px4fmu case (similar to e.g. openpilot_revo).